config/yurt-controller/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
I think it's not a good idea to import cert manager into OpenYurt to manage the certificates for core components. we need to keep the simple and cleanness for OpenYurt.

Reply:
by the way, we had a decision to import a more simpler tool: oamdev/kube-webhook-certgen:v2.4.1 to create or patch certificate for webhooks in OpenYurt community. 
maybe we can use this tool like yurt-app-manager: https://github.com/openyurtio/yurt-app-manager/tree/master/charts/yurt-app-manager/templates/admission-webhooks/job-patch

Reply:
jet/kube-webhook-certgen is not recommended for production, as stated in this project. If a user deletes a certificate by mistake in a production environment, it does not automatically heal